README.md

```diff
@@ -76,7 +76,13 @@ export default defineConfig(({ mode }) => {
   }
 }
 ```
-5. To run locally type `npm run dev`
+5. In _**app.js**_ replace line 2 with `const API_GATEWAY_URL = import.meta.env.GATEWAY_API_URL;`.
```

Comment:
I really think that you shouldn't have to change the code to run in dev and prod. Maybe a if statement somewhere changes between dev and prod.